### PR TITLE
updated libretro and game.libretro packages

### PR DESCRIPTION
**Pull request by CvH**
This is far from optimal (blindly bumping to some random version) but till this is fixed upstream (creating tags/releases...) here is the next round.

it builds at x86_64 ... everything else is untested

### CODE REVIEW

File: packages/emulation/libretro-fbalpha/package.mk [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libretro-fbalpha"
-PKG_VERSION="9ceab7f6293864591e63a2669396dc200e3246bb"
-PKG_SHA256="26eb3e4c3f5c7238fd9632c055d2941be50348fc16998c774126aba2e81508d2"
+PKG_VERSION="20daa807957d7fce64371620271b502811318163"
```
Comment (Raybuntu) [inline packages/emulation/libretro-fbalpha/package.mk]: This hash is no longer available upstream.

Comment (CvH) [reply packages/emulation/libretro-fbalpha/package.mk]: tx fixed

**Comment by CvH** [comment]
Updated ppsspp to 1.7 (that needs the SDL2 +fpic change) and optimized the mkpkg script to save 2gb at the package generating and 60mb at the package itself.

File: packages/emulation/libretro-ppsspp/package.mk [hunk 2]
```diff
@@ -2,26 +2,39 @@
...
 
 PKG_LIBNAME="ppsspp_libretro.so"
 PKG_LIBPATH="lib/$PKG_LIBNAME"
 PKG_LIBVAR="PPSSPP_LIB"
 
-PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=ON"
+if [ "$PROJECT" = "Amlogic" ] || [ "$PROJECT" = "RPi" ]; then
+  case $DEVICE in
+     KVIM|RPi2|S905|Odroid_C2)
+      ARCH_ARM="-DARMV7=ON \
```
Comment (MilhouseVH) [inline packages/emulation/libretro-ppsspp/package.mk]: `PKG_ARCH_ARM`?

Comment (CvH) [reply packages/emulation/libretro-ppsspp/package.mk]: done

**Pull request merged**